### PR TITLE
Newunittestroad

### PR DESCRIPTION
**Pull request by hu-stefan**
# IFC4.3 Unit test

## Unit test
UT Road using IfcOpenCrossProfileDef

## Review requested
- `IfcRoad` 
- `IfcPavement`
- `IfcOpenCrossProfileDef`

for alignment issues -> #66

@pjanck, @seb-esser

### CODE REVIEW

File: IFC 4.3/Road_1/UT_Road_1.ifc [hunk 1]
```diff
@@ -0,0 +1,95 @@
+ISO-10303-21;
+HEADER;
+FILE_DESCRIPTION(('ViewDefinition [Ifc4X3NotAssigned]'),'2;1');
+FILE_NAME(
+/* name */ 'export\\sample_ifcroad.ifc',
```
Comment (seb-esser) [inline IFC 4.3/Road_1/UT_Road_1.ifc:5]:
```suggestion
/* name */ 'UT_Road_1.ifc',
```

File: IFC 4.3/Road_1/UT_Road_1.ifc [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+/* time_stamp */ '2021-01-12T16:51:14',
+/* author */ ('stefan'),
+/* organization */ ('Unknown'),
```
Comment (seb-esser) [inline IFC 4.3/Road_1/UT_Road_1.ifc:8]:
```suggestion
/* organization */ ('TUM Chair of Computational Modeling and Simulation'),
```

File: IFC 4.3/Road_1/readme.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
+
+## Road_1
+
+### Intent
+
+Automatically generated IFC file from a corresponding test file in Civil 3D. 
+
+Converting the data given by the API of Civil into the IFC schema. Exports a
```
Comment (seb-esser) [inline IFC 4.3/Road_1/readme.md:8]:
```suggestion
Converting the data exctracted from a Civil3D project into an IFC model. Exports a
```

File: IFC 4.3/Road_1/readme.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+This scenario builds upon following other scenarios:
+- Alignment_3
+
+
+### Content
+
+This scenario covers the following concepts and/or IFC entities:
+- `IfcRoad` 
+- `IfcPavment`
```
Comment (seb-esser) [inline IFC 4.3/Road_1/readme.md:33]:
```suggestion
- `IfcPavement`
```

**Review by seb-esser — COMMENTED** [review]
minor update requests.

**Comment by pjanck** [comment]
We're moving to the new repository: https://github.com/bSI-InfraRoom/IFC-infra-unit-test

Please open a new pull request there and close this one.

**Pull request closed without merging**